docs/getting-started/continuous_integration.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: default
+title: Continuous Integration
+parent: Getting started
+nav_order: 5
+permalink: /getting-started/continuous-integration/
+---
+
+# Continuous Integration
+
+OSS-Fuzz offers **CIFuzz**, which will run your fuzz targets each time a pull request
+is submitted, for projects hosted on GitHub. This allows you to detect and
+fix bugs before they are submitted.
+
+## How it works
+
+CIFuzz works by checking out a repository at the head of a pull request. The
+project's fuzz targets are built and run for
+a definite amount of time (default is 10 minutes). If a bug is found, the
+stack trace as well as the test case are made abailable for download.
+ If a crash is not found the test passes with a green check.
+
+## Requirements
+
+1. Your project must be integrated in OSS-Fuzz.
+1. Your project is hosted on GitHub.
+
+## Integrating into your repository
+
+You can integrate CIFuzz into your project using the following steps:
+1. Create a `.github` directory in the root of your project.
+1. Create a `workflows` directory inside of your `.github` directory.
+1. Copy the example [`main.yml`](https://github.com/google/oss-fuzz/blob/master/infra/cifuzz/example_main.yml)
+file over from the OSS-Fuzz repository to the `workflows` directory.
+1. Change the `project-name` value in `main.yml` from `example` to the name of your OSS-Fuzz project. It is **very important** that you use your OSS-Fuzz project name which is case sensitive. This name
+is what your project is refered to in the [`projects`](https://github.com/google/oss-fuzz/tree/master/projects) directory of OSS-Fuzz.
```

Comment:
"is the name of your project's subdirectory in the "projects" directory of OSS-Fuzz.

Reply:
Fixed.